build.gradle

```diff
@@ -197,6 +197,8 @@ dependencies {
     // there is a compatibility problem between the Guava Library used by HDFS, Google Cloud Storage and Reflections; the only version that fits for all is the following
     testCompile group: 'com.google.guava', name: 'guava', version: '20.0'
 
+    compile group: 'commons-beanutils', name: 'commons-beanutils', version: '1.9.4'
```

Comment:
Since this is a new dependency for apoc and we don't really depend on it directly, that change should carry a comment like "force 1.9.4 of beanutils as transitive dependency due to CVE xxxx"

Reply:
done
thanks a lot!